### PR TITLE
Exclude helm windows from golden ratio

### PR DESCRIPTION
**Pull request by ralesi**

### CODE REVIEW

File: spacemacs/packages.el [hunk 1]
```diff
@@ -1242,6 +1242,8 @@ Example: (evil-map visual \"<\" \"<gv\")"
                                          "speedbar-mode"
                                          ))
 
+      (add-to-list 'golden-ratio-exclude-buffer-regexp "*[hH]elm.*")
```
Comment (bmag) [inline spacemacs/packages.el]: Shouldn't the regexp be `"^\\*[hH]elm.*"`?

Comment (ralesi) [inline spacemacs/packages.el]: Correct, the package owner changed the regex check after I submitted the patch on that side.  
Thanks.

**Comment by syl20bnr** [comment]
Another great surgical strike from @ralesi :-)
Thank you!
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by synic** [comment]
I'm not sure this patch is working.  I have updated spacemacs, I can see the `golden-ratio-exclude-buffer-regexp` in `~/.emacs.d/spacemacs/packages.el`, however, the helm window still resizes wildly when entering or leaving that window, with golden ratio turned on.

**Comment by syl20bnr** [comment]
I think that depending on the number of candidates the size may vary but it should be remain fixed when narrowing candidates.

**Pull request closed without merging**